pimd/pim_cmd.c

```diff
@@ -8164,7 +8201,8 @@ DEFUN (interface_ip_pim_drprio,
 	nb_cli_enqueue_change(vty, "./dr-priority", NB_OP_MODIFY,
 			      argv[idx_number]->arg);
 
-	return nb_cli_apply_changes(vty, "./frr-pim:pim");
+	return nb_cli_apply_changes(vty, "./frr-pim:pim/address-family[address-family='%s']",
```

Comment:
Let's also add a macro for `"./frr-pim:pim/address-family[address-family='%s']"`